dev/com.ibm.ws.security.oidc.social_fat.backchannelLogout/publish/files/serversettings/client_logoutTokenValidation_variables.xml

```diff
@@ -0,0 +1,26 @@
+<!--
+    Copyright (c) 2022 IBM Corporation and others.
+    All rights reserved. This program and the accompanying materials
+    are made available under the terms of the Eclipse Public License v1.0
+    which accompanies this distribution, and is available at
+    http://www.eclipse.org/legal/epl-v10.html
+   
+    Contributors:
+        IBM Corporation - initial API and implementation
+ -->	
+ <server>
+ 	<variable name="HS384" value="HS256"/>  <!-- setting the sig alg to HS256 so we can login - the test will reset the value to HS384 before the logout is called to valudate the logout_token -->
```

Comment:
I need to create a logout_token that the OP will recognize (sid/sub).  So, I create the id_token using the OP, but, it can't create tokens for HS384, ... So, I use what the OP will support and then the test code builds a logout_token based on the id_token and the signing/encrypting settings that the test needs.  I then update client config to use the value that the test needs to properly validate the token.
I've added more detailed comments to the variable config file.